pkg/collect/postgres_test.go

```diff
@@ -79,7 +79,7 @@ func TestCollectPostgres_createConnectConfigPlainText(t *testing.T) {
 			}
 
 			connCfg, err := c.createConnectConfig()
-			assert.Equal(t, err != nil, tt.hasError)
+			assert.Equal(t, tt.hasError, err != nil)
```

Comment:
This just swaps actual and expected values so its easier to understand test failure results